community/tests/test_git.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+
+from community.git import get_config_remote
+from community.git import get_deploy_url
+from community.git import get_org_name
+from community.git import get_remote_url
```

Comment:
Why not do?
```python
from community.git import (
    get_config_remote,
    get_deploy_url,
    get_org_name,
    ...
)
```

Reply:
That style consumes two extra lines, and depends on ppl remembering to use "trailing commas", which coala dont have a linter for. (isort may be able to enforce this, but isort has other problems)